code/core/src/com/ece454/gotl/Goose.java

```diff
@@ -133,12 +133,20 @@ public boolean isLevelFailed() {
 
     public void jump(Vector2 drag) {
         if (drag.y < 0) return;
+        if (isJumping) return;
```

Comment:
More readable & less spaghetti-code-ish to remove isJumping & call checkIfIdle here directly